src/Events/Integrations/Plugins/Elementor/Controller.php

```diff
@@ -254,6 +256,47 @@ public function suppress_query_filters( $query_args ): array {
 		return $query_args;
 	}
 
+	/**
+	 * Disables the Blocks Editor on posts that have been edited with Elementor.
+	 *
+	 * @since TBD
+	 *
+	 * @param bool $blocks_enabled Whether the Blocks Editor is enabled or not.
+	 *
+	 * @return bool
+	 */
+	public function disable_blocks( $blocks_enabled ) {
+		$post_id = get_the_ID();
+
+		if ( ! $post_id ) {
+			$post_id = Tribe__Main::post_id_helper();
+		}
+
+		if ( ! $post_id ) {
+			$post_id = tribe_get_request_var( 'post' );
+		}
+
+		// We can't get the post ID, bail out.
+		if ( ! $post_id ) {
+			return $blocks_enabled;
+		}
+
+		// Not an event, bail out.
+		if ( ! tribe_is_event( $post_id ) ) {
+			return $blocks_enabled;
+		}
+
+		$elementor_edit = get_post_meta( $post_id, '_elementor_edit_mode', true );
+
+		// Missing the expected meta - bail out.
+		if ( empty( $elementor_edit ) ) {
+			return $blocks_enabled;
+		}
+
```

Comment:
We honestly should have a method to handle this piece of logic, so we can apply to others, there a constant from the Elementor document class we should use;

`Document::BUILT_WITH_ELEMENTOR_META_KEY`